README.md

```diff
@@ -1,8 +1,154 @@
+                                                       ProvarDX command.
+
+  -f, --filespec=filespec                              Specify comma-delimited, ordered paths of test cases to be
+                                                       executed. If not provided all tests in the PROJECTPATH will be
+                                                       executed.
+
+  -l, --loglevel=loglevel                              Specify the level of feedback provided during the compilation
+                                                       (see above) and execution.
+
+  -m, --metadatalevel=metadatalevel                    Specify permitted values reload (get all metadata - default) |
+                                                       refresh (only download changes). This overrides any settings made
+                                                       in the propertyfile.
+
+  -o, --connectionoverridefile=connectionoverridefile  Connection file in the format provardx-connection-schema.json
+                                                       providing a mapping between the Provar project Connection names
+                                                       and the target users to be used.
+
+  -p, --propertyfile=propertyfile                      provardx-properties.json file. If filepath specified the filename
+                                                       will be assumed or it can be specified (.e.g myprops.json). File
+                                                       must be in JSON format and conform to the provardx-properties
+                                                       JSON schema.
+
+  -s, --secrets=secrets                                Specify path to secrets file
+
+  --json                                               format output as json
+
+EXAMPLE
+  $ sfdx provar:runtests -f './myproperties.json'
+```
+
+_See code: [src/commands/provar/runtests.ts](https://github.com/ProvarTesting/provardx/blob/v0.0.1/src/commands/provar/runtests.ts)_
+
+## `sfdx provar:validate [-p <string>] [--json] [-l <string>]`
+
+Validate the provardx property file agains standard JSON Schema
+
+```
+USAGE
+  $ sfdx provar:validate [-p <string>] [--json] [-l <string>]
+
+OPTIONS
+  -l, --loglevel=loglevel          Allow to provide loglevels.
+  -p, --propertyfile=propertyfile  Specify custom property file.
+  --json                           format output as json
+
+EXAMPLES
+  $ sfdx provar:validate
+       Property file is valid.
+    
+  $ sfdx provar:validate --json
+       Invalid property file.
+       [Detailed validation error shown]
+    
+  $ sfdx provar:validate -p './provardx-properties.json' --json
+       Invalid property file.
+       [Detailed validation error shown]
+```
+
+_See code: [src/commands/provar/validate.ts](https://github.com/ProvarTesting/provardx/blob/v0.0.1/src/commands/provar/validate.ts)_
+<!-- commandsstop -->
```

Comment:
Should we reference any documentation or github projects provided by salesforce for sfdx?